df.announcements.xml

```diff
@@ -507,6 +507,15 @@
         <bool name='bright' init-value='true'/>
     </struct-type>
 
+    <struct-type type-name='announcement_alertst'>
+        <enum base-type='int32_t' type-name='announcement_alert_type' name='type'/>
+        <stl-vector name='report_id' type-name='int32_t'/>
+        <stl-vector name='unit_id' type-name='int32_t'/>
+        <stl-vector name='combat_type'>
```

Comment:
bay12 name is `report_unit_announcement_category` (and the vector type is `UnitAnnouncementCategory`)

Reply:
I'm using a pre-existing enum for the category. Doesn't look like `original-name` works with enum, so I've just put a comment on it in `df.units.xml`.